Makefile

```diff
@@ -17,7 +17,7 @@ clean-pyc: ## remove Python file artifacts
 	find . -name '*~' -exec rm -f {} +
 
 lint: ## check style with flake8
-	flake8 django_readonly_field tests
+	flake8 --exclude="migrations" django_readonly_field tests
```

Comment:
it's better to do it in the `setup.cfg`. For example your editor won't complaint about the migrations file.